_includes/specials/policy/en/09-proposals.md

```diff
@@ -0,0 +1,113 @@
+respective fee-bumping children, removing the need for multiple anchor
+outputs on each commitment transaction.
+
+A caveat is that existing RBF rules require the replacement
+transaction to pay a higher absolute fee than the aggregate fees paid
+by all to-be-replaced transactions. This rule helps prevent DoS
+through repeated replacements, but allows a malicious user to increase
+the cost to replace their transaction by attaching a child that is
+high fee but low feerate. This hinders the transaction from being
+mined by unfairly preventing its replacement by a high-feerate
+package, and is often referred to as “Rule 3 pinning.”
+
+Developers have also proposed entirely different ways of adding fees
+to presigned transactions. For example, signing inputs of the
+transaction using `SIGHASH_ANYONECANPAY | SIGHASH_ALL` could allow the
+transaction broadcaster to provide fees by appending additional inputs
+to the transaction without changing the outputs. However, as RBF does
+not have any rule requiring the replacement transaction to have a
+higher “mining score” (i.e. would be selected for a block faster), an
+attacker could pin these types of transactions by creating
+replacements encumbered by low-feerate ancestors. What complicates
+the accurate assessment of the mining score of transactions and
+transaction packages is that the existing ancestor and descendant
+limits are insufficient to bound the computational complexity of this
+calculation. Any transactions it is connected to can influence the
+order in which transactions get picked into a block. A fully-connected
+component, called a _cluster_, can be of any size given current
+ancestor and descendant limits.
+
+A long term solution to address some mempool deficiencies and RBF
+pinning attacks is to [restructure the mempool data structure to track
+clusters][mempool clustering] instead of just ancestor and descendant
+sets. These clusters would be limited in size. A cluster limit would
+restrict the way users can spend unconfirmed UTXOs, but make it
+feasible to quickly linearize the entire mempool using the ancestor
+score-based mining algorithm, build block templates extremely quickly,
+and add a requirement that replacement transactions have a
+higher mining score than the to-be-replaced transaction(s).
+
+Even so, it’s possible that no single set of policies can meet the
+wide range of needs and expectations for transaction relay. For
+example, while batched payments may require a number of unconfirmed
+descendants, this leaves room for pinning package RBF of a shared
+transaction through absolute fees. A proposal for [v3 transaction
+relay policy][topic v3 transaction relay] was developed to allow
+contracting protocols to opt in to a more restrictive set of package
+limits. V3 transactions would only permit packages of size two (one
+parent and one child) and limit the weight of the child. These limits
+would mitigate RBF pinning through absolute fees, and offer some of
```

Comment:
```suggestion
would delimit RBF pinning through absolute fees, and offer some of
```

Reply:
I don't see a problem with "mitigate" and I think it's better than "these limits would delimit [...]".